components/apimgt/org.wso2.carbon.apimgt.rest.api/src/gen/java/org/wso2/carbon/apimgt/rest/api/model/API.java

```diff
@@ -284,6 +309,8 @@ public String toString()  {
     sb.append("  swagger: ").append(swagger).append("\n");
```

Comment:
Please note that these files get auto-generated using the swagger-to-cxf maven plugin we use for REST API development.